v20.2/create-database.md

```diff
@@ -25,6 +25,7 @@ Parameter | Description
 `IF NOT EXISTS` | Create a new database only if a database of the same name does not already exist; if one does exist, do not return an error.
 `name` | The name of the database to create, which [must be unique](#create-fails-name-already-in-use) and follow these [identifier rules](keywords-and-identifiers.html#identifiers).
 `encoding` | The `CREATE DATABASE` statement accepts an optional `ENCODING` clause for compatibility with PostgreSQL, but `UTF-8` is the only supported encoding. The aliases `UTF8` and `UNICODE` are also accepted. Values should be enclosed in single quotes and are case-insensitive.<br><br>Example: `CREATE DATABASE bank ENCODING = 'UTF-8'`.
+`CONNECTION LIMIT` | <span class="version-tag">New in v20.2:</span> Supported for compatibility with PostgreSQL. By default, CockroachDB does not limit connections, and `CONNECTION LIMIT = -1`. This clause is a no-op for the default value (`-1`), and all other values return an error.
```

Comment:
nit: ```` , and `CONNECTION_LIMIT=1` ``` is worded in a way hard to read. Maybe `A value of -1 indicates no limit. By default, CockroachDB sets `CONNECTION LIMIT = -1`

Reply:
TFTR, @otan. You're right - it was a little lacking in clarity. Great suggestions. Updated.